draft-ietf-teep-architecture.md

```diff
@@ -354,9 +354,9 @@ reducing access to keying material.
 ## Internet of Things
 
 Weak security in Internet of Things (IoT) devices has been posing threats to 
-critical infrastructure that relies upon such devices.
-It is desirable that IoT devices can prevent malware from
-manipulating actuators (e.g., unlocking a door), or
+critical infrastructure, i.e. assets that are essential for the functioning
```

Comment:
i.e. should always have a comma following it, per Chicago Manual of Style (which the RFC editor follows)

Reply:
Fixed in a new PR 238.